src/declarations/stencil-public-compiler.ts

```diff
@@ -288,15 +288,6 @@ export interface ConfigExtras {
    */
   __deprecated__cssVarsShim?: boolean;
 
-  // TODO(STENCIL-661): Remove code related to the dynamic import shim
-  /**
-   * Dynamic `import()` shim. This is only needed for Edge 18 and below, and
-   * Firefox 67 and below. Defaults to `false`.
-   * @deprecated Since Stencil v3.0.0. IE 11, Edge <= 18, and old Safari
-   * versions are no longer supported.
-   */
-  __deprecated__dynamicImportShim?: boolean;
```

Comment:
There's a few references to the old property name I found in some of our tests. Can we remove these while we're in here?

I found a few references to the _old_ name (`cssVarsShim`, prior to the rename) in:

- https://github.com/ionic-team/stencil/blob/42314dc202e9832086207759565c6660d3746eff/test/jest-spec-runner/stencil.config.ts#L13
- https://github.com/ionic-team/stencil/blob/42314dc202e9832086207759565c6660d3746eff/test/todo-app/stencil.config.ts#L15
- https://github.com/ionic-team/stencil/blob/42314dc202e9832086207759565c6660d3746eff/test/karma/stencil.config.ts#L35